README.md

```diff
@@ -1,623 +1,1519 @@
 
   
 
+  
+
 >  **Caution:** While the default values in the downloaded environment files enable the Project Application to operate, certain features may not function correctly or could be impaired unless the adopter-specific environment variables are properly configured.
 
   
 
+  
+
 4.  **Download `replace_volume_path` Script File**
 
   
 
+  
+
 -  **Ubuntu/Linux/Mac**
 
-```
+  
 
+```
 curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/1.0.0/dockerized/scripts/mac-linux/replace_volume_path.sh
-
 ```
-<!--
-
-- **Windows**
-
+-  **Windows**
 ```
-
 curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/1.0.0/dockerized/scripts/windows/replace_volume_path.bat
-
 ```
-
--->
+5.  **Run `replace_volume_path` Script File**
 
   
 
-5.  **Run `replace_volume_path` Script File**
-
   
 
 -  **Ubuntu/Linux/Mac**
 
+  
+
 1. Make the `replace_volume_path.sh` file an executable.
 
-```
+  
 
+```
 chmod +x replace_volume_path.sh
-
 ```
 
+  
+
 2. Run the script file using the following command.
 
-```
+  
 
+```
 ./replace_volume_path.sh
-
 ```
 
   
-  
-  
+-  **Windows**
+  1. Run the script file using the following command.
+  ```
+replace_volume_path.sh
+```
 
 6. ** Download `docker-compose-up` & `docker-compose-down` Script Files **
 
   
 
+  
+
 -  **Ubuntu/Linux/Mac**
 
   
 
-1. Download the files.
+  
 
-```
+1. Download the files.
 
-curl -OJL https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/scripts/mac-linux/docker-compose-up.sh
+  
 
 ```
+curl -OJL https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/scripts/mac-linux/docker-compose-up.sh
 ```
 
+```
 curl -OJL https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/scripts/mac-linux/docker-compose-down.sh
-
 ```
-2. Make the files executable by running the following commands.
 
-```
+2. Make the files executable by running the following commands.
 
-chmod +x docker-compose-up.sh
+  
 
 ```
+chmod +x docker-compose-up.sh
 ```
 
+```
 chmod +x docker-compose-down.sh
-
 ```
-<!-- - **Windows**
 
-```
+ -  **Windows**
 
-curl -OJL https://github.com/ELEVATE-Project/samiksha-service/raw/features_dockerSetup/documentation/1.0.0/dockerized/scripts/windows/docker-compose-up.bat
+  
 
 ```
+curl -OJL https://github.com/ELEVATE-Project/samiksha-service/raw/features_dockerSetup/documentation/1.0.0/dockerized/scripts/windows/docker-compose-up.bat
 ```
 
+```
```

Comment:
@borkarsaish65 please check and do this for the entire file. It's not aligned properly and unwanted lines are there. Please correct all.

Reply:
corrected now